pykafka/simpleconsumer.py

```diff
@@ -260,7 +260,8 @@ def _handle_NotLeaderForPartition(parts):
             OffsetOutOfRangeError.ERROR_CODE: _handle_OffsetOutOfRangeError,
             NotLeaderForPartition.ERROR_CODE: _handle_NotLeaderForPartition,
             OffsetMetadataTooLarge.ERROR_CODE: lambda p: raise_error(OffsetMetadataTooLarge),
-            NotCoordinatorForConsumer.ERROR_CODE: _handle_NotCoordinatorForConsumer
+            NotCoordinatorForConsumer.ERROR_CODE: _handle_NotCoordinatorForConsumer,
+            GroupLoadInProgress.ERROR_CODE: lambda p: raise_error(GroupLoadInProgress)
```

Comment:
I don't think you want `raise_error` here - essentially this just needs to be a noop.